doc/classes/Parallax2D.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="Parallax2D" inherits="Node2D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A node used to create a parallax scrolling background.
+	</brief_description>
+	<description>
+		A [Parallax2D] is used to create a parallax effect. It can move at a different speed relative to the camera movement using [member scroll_scale]. This creates an illusion of depth in a 2D game. If manual scrolling is desired, the [Camera2D] position can be ignored with [member ignore_camera_scroll].
+		[b]Note:[/b] Any changes to this node's position made after it enters the scene tree will be overridden if [member ignore_camera_scroll] is [code]false[/code] or [member screen_offset] is modified.
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="autoscroll" type="Vector2" setter="set_autoscroll" getter="get_autoscroll" default="Vector2(0, 0)">
+			Velocity at which the offset scrolls automatically, in pixels per second.
+		</member>
+		<member name="follow_viewport" type="bool" setter="set_follow_viewport" getter="get_follow_viewport" default="true">
+			If [code]true[/code], this [Parallax2D] is offset by the current camera's position. It may be desired to disable it if the [Parallax2D] is in a [CanvasLayer] separate from the current camera.
```

Comment:
It might be worth it to explicitly say that `follow_viewport` should match the `follow_viewport` of CanvasLayer if using one.

Reply:
I was hesitant to say this (because who knows how the user wants to use it), but since documentation is for those who are unfamiliar, trying to understand and getting it to work, I think you're right to just explicitly suggest a scenario for when it's most likely desired. Thanks!